ankdown/ankdown.py

```diff
@@ -210,11 +213,18 @@ def __getitem__(self, deckname):
         return super(DeckCollection, self).__getitem__(deckname)
 
 def field_to_html(field):
-    # Need to extract the math in brackets so that it doesn't get
-    # markdowned.
-    for bracket in ["(", ")", "[", "]"]:
-        field = field.replace(r"\{}".format(bracket), r"\\{}".format(bracket))
-        # backslashes, man.
+    """Need to extract the math in brackets so that it doesn't get markdowned.
+    If math is separated with dollar sign it is converted to brackets."""
+    if FLAG_DOLLAR:
+        for sep_dollar, sep_bracket in zip(["$", "$$"], [[r"\\(", r"\\)"], [r"\\[", r"\\]"]]):
```

Comment:
Does this code leave any way to enter a literal dollar sign?

Reply:
I must admit I didn't thought about the implementation much. Thank you for your time pointing this out.

So we need a way to escape this. Typically this would be a backslash in front.
Maybe a simple `replace` doesn't work anymore, and we have to use a regex pattern, that avoid splitting at points with `\$`.

I will come up with something in the next few days.